src/components/signin/Signin.js

```diff
@@ -0,0 +1,84 @@
+// @flow
```

Comment:
it seems you just coppied signinstate to signin. we dont need. signupstate was a wizard. signin is not a wizard we dont need it. its just one simple route. signininfo

Reply:
just have SigninInfo added to the signuprouter

Reply:
done